### PR TITLE
Add package definition for JunctionSeq 0.4.19

### PR DESCRIPTION
**Pull request by davebx**
Part of the resolution for #355

### CODE REVIEW

File: packages/package_junctionseq_0_4_19/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                        <package name="R" version="3.2.1" />
+                    </repository>
+                    <!-- ?statmod?, ?plotrix?, ?stringr?, ?Biobase?, ?locfit?, ?Rcpp?, ?RcppArmadillo?, ?BiocGenerics?, ?BiocParallel?, ?genefilter?, ?geneplotter?, ?ggplot2?, ?Hmisc?, ?S4Vectors?, ?IRanges?, ?GenomicRanges? -->
+                    <package>https://github.com/bgruening/download_store/raw/master/DEXSeq_1.14.2/statmod_1.4.21.tar.gz</package>
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-10-29/src/contrib/plotrix_3.6.tar.gz</package>
+                    <package>https://mran.revolutionanalytics.com/snapshot/2015-10-29/src/contrib/stringi_1.0-1.tar.gz</package>
+                    <package>https://github.com/bgruening/download_store/raw/master/DESeq2-1_8_1/magrittr_1.5.tar.gz</package>
+                    <package>https://github.com/bgruening/download_store/raw/master/DESeq2-1_8_1/stringr_1.0.0.tar.gz</package>
+					<package>https://bioarchive.galaxyproject.org/BiocGenerics_0.17.0.tar.gz</package>
+					<package>https://bioarchive.galaxyproject.org/Biobase_2.29.1.tar.gz</package>
```
Comment (erasche) [inline packages/package_junctionseq_0_4_19/tool_dependencies.xml]: indentatiob

**Comment by erasche** [comment]
Otherwise looks OK.

File: packages/package_junctionseq_0_4_19/tool_dependencies.xml (hunk 1 above)
Comment (davebx) [inline packages/package_junctionseq_0_4_19/tool_dependencies.xml]: Oh for...

**Comment by martenson** [comment]
awesome

File: packages/package_junctionseq_0_4_19/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads and installs version
+  0.4.19 of the JunctionSeq R library.
+long_description: |
+  The JunctionSeq R package is a powerful tool for testing differential junction usage
+  (DJU) in next-generation, high-throughput RNA-Seq experiments.
+  https://github.com/hartleys/JunctionSeq
```
Comment (martenson) [inline packages/package_junctionseq_0_4_19/.shed.yml]: can we please put this as `homepage_url`?

**Comment by erasche** [comment]
testing urls now, will merge in a sec.

File: packages/package_junctionseq_0_4_19/.shed.yml (hunk 2 above)
Comment (erasche) [inline packages/package_junctionseq_0_4_19/.shed.yml]: xref https://github.com/galaxy-iuc/standards/issues/1

Comment (davebx) [inline packages/package_junctionseq_0_4_19/.shed.yml]: Yes.

**Comment by martenson** [comment]
published to the Tool Shed: https://toolshed.g2.bx.psu.edu/view/iuc/package_junctionseq_0_4_19/4a085ebd7061

**Comment by bgruening** [comment]
Thanks Martin!
@galaxyproject/iuc as a general question do we want to upload those packages to the MTS when we do not have anyone that is using it yet?

**Pull request merged**